entgql/annotation.go

```diff
@@ -173,6 +201,16 @@ func (a *Annotation) Decode(annotation interface{}) error {
 	return json.Unmarshal(buf, a)
 }
 
+// Any returns true if the skip annotation is setted
+func (f SkipMode) Any() bool {
+	return f != 0
+}
+
+// Has check if the skip annotation has a specific flag
+func (f SkipMode) Has(check SkipMode) bool {
+	return f&check == check
+}
```

Comment:
```suggestion
// Is checks if the skip annotation has a specific flag.
func (f SkipMode) Is(mode SkipMode) bool {
	return f&mode != 0 
}
```

ent uses `Is` in multiple places for big flags.